packages/blocks/src/lib/maplibre/index.svelte

```diff
@@ -0,0 +1,129 @@
+<!--
+  @component
+
+  Creates a maplibre-gl map that will fill its parent.
+
+  Children will mount once the map is fully loaded.
+
+  ```svelte
+    <MapLibre>
+      <MapLibreMarker lngLat={{ lng: 0, lat: 0 }} />
+    </MapLibre>
+  ```
+-->
+<script lang='ts'>
```

Comment:
Should we be adding?

```svelte
<svelte:options immutable />
```

Reply:
Done!

Reply:
Are we sure? `map` looks to be a mutable value that we use in reactive statements

Reply:
Map isn't a prop though?

Reply:
Pretty sure the immutable flag is related to props only, check this out:

https://svelte.dev/repl/6b600f4495b14e2f92555cd6b062de5c?version=4.2.0

Toggling the flag in the "Greeting" component will determine whether assignments to nested properties will reflect in reactive updates in children. If you look at the compiler output it's just swapping a `safe_not_equal` | `not_equal` function being piped into that component